src/server/pps/server/api_server.go

```diff
@@ -2745,6 +2710,21 @@ func (a *apiServer) RunPipeline(ctx context.Context, request *pps.RunPipelineReq
 	return nil, errors.New("unimplemented")
 }
 
+func commitFilesetInTransaction(a pfsServer.APIServer, txnCtx *txncontext.TransactionContext, branch *pfs.Branch, filesetID string) (*pfs.Commit, error) {
```

Comment:
Functions, variables, etc that reference file sets should be ...FileSet or fileSet... since file set is two words rather than one now.